src/ralph/cli.py

```diff
@@ -314,7 +314,7 @@ def auth(
     username,
     password,
     scope,
-    write,
+    write_credentials,
```

Comment:
I don't think it's required.

Reply:
since the function is also named `write`, this was done to conform to linter.

Reply:
What about calling it `write_to_disk`?